PORTAL/jobs/__init__.py

```diff
@@ -123,12 +134,8 @@ def fs(self) -> _utils.FSTree:
         return self._fs.jobs.copy()
 
     @property
-    def queue(self):
-        try:
-            return self._queue
-        except AttributeError:
-            self._queue = queue_mod.JobQueue.from_fstree(self._fs.queue)
-            return self._queue
+    def queues(self) -> queue_mod.JobQueues:
+        return queue_mod.JobQueues.from_config(self._cfg, self._fs.queues)
```

Comment:
We may be grateful later if we cache the value:

```suggestion
    def queues(self) -> queue_mod.JobQueues:
        try:
            return self._queues
        except AttributeError:
            self._queues = queue_mod.JobQueues.from_config(self._cfg, self._fs.queues)
            return self._queues
```